### ISSUE TITLE
[A11Y] [VoiceOver] - unable to select dropdown items with voiceover

### ISSUE THREAD

**Issue by nadokupa745** [OfficeDev/office-ui-fabric-react#11247]
### Environment Information

- **Package version(s)**: ("^6.124.2",)
- **Browser and OS versions**: (Safari and IOS 12.4.3)
-** Device: Iphone

### Describe the issue:
VoiceOver -
Focus is not set to the dropdown options on expanding and hence cannot use voiceover for selecting/navigating dropdown items

#### Actual behavior:
Focus moves to elements outside the drop down list while navigating dropdown items in forward direction, unable to select dropdown items

#### Expected behavior:
Focus should move to the first item in the list when user navigates in forward direction on expanding the dropdown.

**Comment by aneeshack4** [comment]
Does not repro on Mac or iOS:

![image](https://user-images.githubusercontent.com/4161791/69198904-273b2000-0aeb-11ea-9e8b-d1c81398618b.png)

![image](https://user-images.githubusercontent.com/4161791/69198975-69646180-0aeb-11ea-8b39-d041c701acb0.png)

**Comment by nadokupa745** [comment]
@aneeshack4 , I had reproduced in mobile device.
Can you pls check on mobile.

**Comment by aneeshack4** [comment]
@nadokupa745 The 2nd screenshot is on mobile - the latest version of iOS + Safari. Please make sure you're using the latest versions of Fabric & the OS on your Apple devices.

**Comment by nadokupa745** [comment]
Upgrading to latest fixed the issue.

**Comment by Ajaz4193** [comment]
I'm able to reproduce the same issue with latest updates
Device: iPhone XR
iOS Version: 13.3
Environment: [https://developer.microsoft.com/en-us/fabric#/controls/web/dropdown](url)

Issue is once the dropdown is activated the list UI is displayed but the focus still is at the background control and user has to explicitly touch the list UI to navigate to the list elements.
Note: Same issue is not reproducible when the screen is in landscape mode.



![IMG_0003](https://user-images.githubusercontent.com/40019907/71298362-9307e700-233c-11ea-8e1d-c2d8dcfd0885.PNG)

**Comment by aneeshack4** [comment]
So I'm able to repro it but it works fine in landscape mode only not in portrait mode which leads me to think that this is due to the dropdowns' width being too small for screen sizes below a certain width causing the focus to be messed up. Tagging component owner @joschect - do you think we should fix this by making the changing the width of the examples or is there something we can change in the component itself?

**Comment by joschect** [comment]
@aneeshack4 I'm not sure about this. As far as I can tell focus is going to correctly to the panel. This is further enforced by the fact that this appears to work in narrator in a small window using arrow keys. I suspect that this is an ios/VO problem but I haven't been able to verify.

**Comment by aneeshack4** [comment]
I believe this is an iOS issue as it doesn't repro on edge + narrator or safari + voiceover: 
![image](https://user-images.githubusercontent.com/4161791/73795637-f5d40700-475f-11ea-8918-3e0bb092d78c.png) so resolving as external.
